src/SFA.DAS.ApplyService.Web/Controllers/ApplicationController.cs

```diff
@@ -565,9 +570,16 @@ public async Task<IActionResult> Submit(Guid applicationId, int sequenceId)
             {
                 return RedirectToAction("Sequence", new { applicationId });
             }
-            
-            await _apiClient.Submit(applicationId, sequenceId, User.GetUserId(), User.GetEmail());
-            return RedirectToAction("Submitted", new {applicationId});
+
+            if (await _apiClient.Submit(applicationId, sequenceId, User.GetUserId(), User.GetEmail()))
+            {
+                return RedirectToAction("Submitted", new { applicationId });
+            }
+            else
+            {
+                // unable to submit
+                return RedirectToAction("NotSubmitted", new { applicationId });
+            }
```

Comment:
Are there any tests that can cover these new rules in this controller?

Reply:
Added unit tests to the handler (which pass this on the result to the controller)